website/docs/cli/state/index.html.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: "docs"
 page_title: "Manipulating State - Terraform CLI"
+description: "State tracks which real-world object corresponds to each resource in the configuration. Inspect state, move or import resources, and more."
```

Comment:
It might be me/nitpicky, but I read "state tracks" as a noun instead of "Terraform state" tracks. can we rephrase the first sentence?

Reply:
I can see that - I'll change it to "State data" which matches how it's referenced in the actual page!